data/databases.yaml

```
@@ -0,0 +1,52 @@
+- key: bdnb
+  name: BDNB
+  url: https://www.data.gouv.fr/fr/datasets/base-de-donnees-nationale-des-batiments/
+  published_by: le CSTB
+  licence: Partiellement ouverte
+  tags: [Pérformance énergétique, DPE, DVF]
+  description: >
+    La BDNB est une cartographie du parc de bâtiments existants. Structurée à la maille « bâtiment », elle contient une carte d’identité pour chacun des 32 millions de bâtiments, résidentiels ou tertiaires. Elle est construite par croisement d’une vingtaine de base de données issues d’organismes publics.
+  image: bdnb.png
+  featured: true
+  featured_summary: La BDNB possèdes des dizaines d'attributs métier issus d'une vingtaine de bases publiques. Elle couvre 32 millions de bâtiments résidentiels et tertiaires.
+- key: bdtopo
+  name: BD Topo
+  url: https://geoservices.ign.fr/bdtopo
+  published_by: l'IGN
+  licence: Données ouvertes
```

```suggestion
  licence: Licence Ouverte / Open Licence version 2.0 
```

Est-ce qu'on mettrait la vraie licence ?

---

Je pense que c'est plus parlant comme ça, non ? C'est pas affiché avec le mot licence